backend/package.json

```diff
@@ -35,6 +36,7 @@
     "typescript": "^5.0.0"
   },
   "dependencies": {
+    "big.js": "^6.2.1",
```

Comment:
BigInt division work too well which affects the precision of cryptocurrencies with 18 decimals (e.g. eth, matic)

Big.js is a lightweight library for BigDecimals since javascript does not have it natively